### PR TITLE
Final draft of MSBR online reprocessing summary

### PR DESCRIPTION
**Pull request by andrewryh**
I have added confidence interval for K-eff, spectrum now is normalized by neutron lethargy, and some minor fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,5 @@
...
-----------------
-Some of these branches will hold templates for certain common formats. These 
-branches will be called 'template-name' such as 'template-ieee' or similar. 
-Please add templates as you discover them.
-
-Rather than duplicate them here, some other formats are found in their
-own repo:
-
-* [ANS Transactions](https://github.com/sethrj/anstrans) by Seth Johnson (@sethrj)
+ANS meeting summary about simulation online reprocessing using Serpen2 capabilities
```
Comment (katyhuff) [inline README.md]: Serpent

File: acros.aux [hunk 2]
```diff
@@ -0,0 +1,24 @@
+\relax
```
Comment (katyhuff) [inline acros.aux]: don't commit aux files.

File: msbr_reproc.tex [hunk 3]
```diff
@@ -157,17 +157,17 @@ \section{Description of the actual work}
...
+periodic. The density of the fuel salt is 3.3304 g/cm$^3$ and the graphite density is 
+1.843 g/cm$^3$. The temperature of fuel and graphite is fixed at 908K for the whole 
 reprocessing cycle \cite{robertson_conceptual_1971}.
 
 \begin{figure}[htbp!] % replace 't' with 'b' to force it to be on the bottom
         \centering
-        \includegraphics[width=0.90\linewidth]{zone_I_mesh.png}
-        \caption{MSBR unit cell of Zone I geometry.}
+        \includegraphics[width=0.95\linewidth]{zone_I_mesh.png}
+        \caption{MoltenSalt Breeder Reactor (MSBR) unit cell of Zone I geometry.}
```
Comment (katyhuff) [inline msbr_reproc.tex]: Molten Salt Breeder Reactor

Comment (katyhuff) [reply msbr_reproc.tex]: (or, you could use "\glsfirst{MSBR}" which will expand it like it's the first time.)

File: msbr_reproc.tex [hunk 4]
```diff
@@ -228,23 +228,22 @@ \subsection{Equilibrium state analysis}
 initial multiplication factor during depletion calculation is selected for 
 a state with fully withdrawn control rods, which gives 
 considerable excess reactivity in the beginning of the cycle (approximately 5000 pcm). 
-Standard deviation for these calculations is approximately 100 pcm.
+Standard deviation for these calculations is approximately 100 pcm. Figure~\ref{fig:keff}
```
Comment (katyhuff) [inline msbr_reproc.tex]: The standard deviation...

**Review by katyhuff — APPROVED** [review]
minor comments

**Comment by andrewryh** [comment]
Thanks!

**Pull request merged**